lib/portlayer/storage/volume_cache.go

```diff
@@ -216,6 +216,21 @@ func (v *VolumeLookupCache) Export(op trace.Operation, store *url.URL, id, ances
 	return vs.Export(op, store, id, ancestor, spec, data)
 }
 
+func (v *VolumeLookupCache) Impport(op trace.Operation, store *url.URL, id string, spec *archive.FilterSpec, tarStream io.ReadCloser) error {
+	storeName, err := util.VolumeStoreName(store)
```

Comment:
Nope need to do the check here before we attempt to actually get the volume store. Since it may be malformed going into the cache.